Makefile

```diff
@@ -255,7 +255,7 @@ GITBASE ?= "https://github.com"
 
 ${GOPATH}/src/istio.io/istio:
 	mkdir -p ${GOPATH}/src/istio.io
-	git clone ${GITBASE}/istio/istio.git ${GOPATH}/src/istio.io/istio
+	git clone -b br_support_isolated_namespace_for_test --single-branch ${GITBASE}/morvencao/istio.git ${GOPATH}/src/istio.io/istio
```

Comment:
can you exclude this change in your commit

Reply:
This PR depends on https://github.com/istio/istio/pull/13830, once https://github.com/istio/istio/pull/13830 is merged, I have to update this to master repo.

Reply:
This is ready to go now